docs/framework/get-started/system-requirements.md

```diff
@@ -26,6 +22,8 @@ manager: "wpickett"
 
 The tables in this topic provide the hardware, operating system, and software requirements for the .NET Framework 4.5 and its point releases (4.5.1 and 4.5.2), the [!INCLUDE[net_v46](../../../includes/net-v46-md.md)] and its point releases (4.6.1 and 4.6.2), and the .NET Framework 4.7. Development environments that enable you to develop apps for the .NET Framework have a separate set of requirements.
 
+All .NET Framework versions since .NET Framework 4 are in-place updates. In other words, if there's a later version installed on the machine already, you don't have to install the previous versions. Similarly, if you install a later version, you don't have to first uninstall the previous versions. And, if an OS already comes pre-installed with a particular .NET Framework version, you cannot install previous 4.x versions on the same machine.
```

Comment:
Sounds good. I'll give it a try and even I was making that confusion. so adding this note I think it will help. And yes, there is an important note.
I'll add the text in an include file so it can be easily added to topics and we'll see if it supports bullet lists. I'm not sure.